corpus/templates/pages/impulse.html

```diff
@@ -0,0 +1,623 @@
+                                fill="currentColor"
+                                />
+                        </svg>
+                    </div>
+                    <div class="w-full">
+                        <h4
+                            class="mt-1 text-lg font-semibold text-dark dark:text-white"
+                            >
+                            Is there a registration fee?
+                        </h4>
+                    </div>
+                    </button>
+                    <div x-show="openFaq2" class="faq-content pl-[62px]">
+                    <p
+                        class="py-3 text-base leading-relaxed text-body-color dark:text-dark-6"
+                        >
+                        <ol class="list-decimal list-inside">
+                            <li>No registration fees for NITK students</li>
+                            <li>No registration fees for teams with at least one IEEE member</li>
+                            <li>A registration fee of Rs.150 for teams with all non-NITK or non-IEEE members</li>
+                        </ol>
+                    </p>
+                    </div>
+                </div>
+                <div
+                    class="w-full p-4 mb-8 bg-base-100 rounded-lg shadow-[0px_20px_95px_0px_rgba(201,203,204,0.30)] dark:shadow-[0px_20px_95px_0px_rgba(0,0,0,0.30)] sm:p-8 lg:px-6 xl:px-8"
+                    >
+                    <button
+                    class="flex w-full text-left faq-btn"
+                    @click="openFaq3 = !openFaq3"
+                    >
+                    <div
+                        class="bg-primary/5 dark:bg-white/5 text-primary mr-5 flex h-10 w-full max-w-[40px] items-center justify-center rounded-lg"
+                        >
+                        <svg
+                            :class="openFaq3 && 'rotate-180'"
+                            width="22"
+                            height="22"
+                            viewBox="0 0 22 22"
+                            fill="none"
+                            xmlns="http://www.w3.org/2000/svg"
+                            >
+                            <path
+                                d="M11 15.675C10.7937 15.675 10.6219 15.6062 10.45 15.4687L2.54374 7.69998C2.23436 7.3906 2.23436 6.90935 2.54374 6.59998C2.85311 6.2906 3.33436 6.2906 3.64374 6.59998L11 13.7844L18.3562 6.53123C18.6656 6.22185 19.1469 6.22185 19.4562 6.53123C19.7656 6.8406 19.7656 7.32185 19.4562 7.63123L11.55 15.4C11.3781 15.5719 11.2062 15.675 11 15.675Z"
+                                fill="currentColor"
+                                />
+                        </svg>
```

Comment:
If the same SVG is used everywhere, put it in a file in `static` and include it here.

Reply:
Extracted all repeated SVGs into `static` "img/symbols"